lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -68,11 +68,8 @@ def define
         option :widow_pension
         option :widows_aged
 
-        next_node_calculation :benefit_related_questions do |response|
-          questions = response.split(",").map { |r| :"#{r}_amount?" }
-          questions << :housing_benefit_amount? if housing_benefit == 'yes'
-          questions << :single_couple_lone_parent?
-          questions
+        next_node_calculation :benefit_types do |response|
+          response.split(",").map(&:to_sym)
```

Comment:
I did consider it, but I thought that having the `if housing_benefit == 'yes'` condition in that `next_node_calculation` block, rather than in each of the `next_node` blocks was harder to follow. Although it results in more duplication, I quite like the explicitness of having it in the `next_node` blocks.